src/Compilers/Core/Portable/Operations/IConditionalGotoStatement.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a conditional goto.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IConditionalGotoStatement : IOperation
+    {
+        /// <summary>
+        /// Condition of the branch.
+        /// </summary>
+        IOperation Condition { get; }
+        /// <summary>
+        /// Label that is the target of the branch.
+        /// </summary>
+        ILabelSymbol Target { get; }
+        /// <summary>
+        /// Jump if the condition is true.
```

Comment:
This is unclear. What does it do if this is `false` and the condition is true?

Reply:
I guess it won't jump in that case. I just copied that property from BoundConditionalGoto.JumpIfTrue property. I think it means if the conditional goto branch is behaving like JE (jumpe on equals) or JNE (jump on not equals). So, if the property JumpIfTrue is false, it will only jump when the condition is false.